File: app/src/main/java/com/example/mixit/MainActivity.java
package com.example.mixit;

import android.content.Intent;
import android.os.Bundle;
import android.view.View;
import android.widget.Button;

import androidx.activity.EdgeToEdge;
import androidx.appcompat.app.AppCompatActivity;
import androidx.core.graphics.Insets;
import androidx.core.view.ViewCompat;
import androidx.core.view.WindowInsetsCompat;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);


        Button button_Start = (Button) findViewById(R.id.StartButton);

        button_Start.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                Intent myIntent = new Intent(MainActivity.this,
                        SignIn.class);
                startActivity(myIntent);
            }
        });

    };

}

File: app/src/main/java/com/example/mixit/MainActivity.kt
package com.example.mixit
import android.os.Bundle
import androidx.activity.ComponentActivity
import androidx.activity.compose.setContent
import androidx.activity.enableEdgeToEdge
import androidx.compose.material3.Text
import androidx.compose.runtime.Composable
import androidx.compose.ui.Modifier
import androidx.compose.ui.tooling.preview.Preview
import com.example.mixit.login.LogIn
import com.example.mixit.ui.theme.MixItTheme

class MainActivity : ComponentActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        enableEdgeToEdge()
        setContent {
            WelcomeScreen()

        }

        }
    }


@Composable
fun Greeting(name: String, modifier: Modifier = Modifier) {
    LogIn()
}

@Preview(showBackground = true)
@Composable
fun GreetingPreview() {
    MixItTheme {
        Greeting("Android")
    }
}